weed/shell/shell_liner.go

```diff
@@ -46,8 +46,8 @@ func RunShell(options ShellOptions) {
 
 	commandEnv := NewCommandEnv(&options)
 
-	go commandEnv.MasterClient.KeepConnectedToMaster()
-	commandEnv.MasterClient.WaitUntilConnected()
+	go commandEnv.MasterClient.KeepConnectedToMaster(context.Background())
+	commandEnv.MasterClient.WaitUntilConnected(context.Background())
```

Comment:
should these two calls share the same context?

Reply:
The Background context is used, so nothing should break. I can make a general one if context.WithCancel is needed in the future